component-samples/demo/README.MD

```diff
@@ -0,0 +1,233 @@
+```
+
+* Use the following command to delete all the podman artifacts. (**Note:** podman containers must be stopped before deleting them)
+```
+podman system prune -a
+```
+# Configuring Proxies
+Update the proxy information in `_JAVA_OPTIONS` as
+
+```
+_JAVA_OPTIONS=-Dhttp.proxyHost=http_proxy_host -Dhttp.proxyPort=http_proxy_port -Dhttps.proxyHost=https_proxy_host -Dhttps.proxyPort=https_proxy_port
+```
+
+where
+
+`http_proxy_host`: Represents the HTTP proxy hostname. Typically, it is an IP address or domain name in the proxy URL.
+
+`http_proxy_port`: Represents the HTTP proxy port. Typically, it is the port number in the proxy URL.
+
+`https_proxy_host`: Represents the HTTPS proxy hostname. Typically, it is an IP address or domain name in the proxy URL.
+
+`https_proxy_port`: Represents the HTTPS proxy port. Typically, it is the port number in the proxy URL.
+
+Specify the combination of the hostname and the port information together for either HTTP, HTTPS, or both. For example, if the HTTP proxy is 'http://myproxy.com:900', then the following updates will be made to the properties:
+
+http_proxy_host: myproxy.com
+
+http_proxy_port: 900
+
+If no proxy needs to be specified, do not add these properties to your _JAVA_OPTIONS.
+
+# Configuring OnDie (Optional)
+
+OnDie is a type of device that makes use of the MAROE prefix. If you need to support such devices then you will need to configure the FDO PRI demo components by adding/updating the following properties. The values can be specified via Java -Doptions, entries in application.properties or entries in the .env files.
+
+OnDie requires several certificates and CRLs. 
+
+`ondie_cache`: (required if supporting OnDie, optional otherwise) Specifies the path to the directory containing the OnDie certificates and CRLs.
+
+
+Finally, the `ondie_cache` directory needs to be copied into the docker/podman container, uncomment the following line in `Dockerfile/Podmanfile` of Manufacturer and Owner.
+```
+COPY ./ondie_cache ./ondie_cache/
+```
+
+
+# Configuring log messages
+
+The components use LOG4J2 for logging. The logging configuration can be updated through log4j2.xml
+file stored within each component folder. By default, INFO logs are prints on the console and DEBUG logs
+are printed in the log file.
+
+Use following command to extract the log file from a running container.
+```
+docker container cp <container-id>:/home/fdo/log-filename .
+```
+
+# Running Demo
+
+***NOTE***: Use the following commands to enbale FDO support on RHEL.
+```
+bash scripts/enable_rhel_support.sh
+echo $'\nexport PODMAN_USERNS=keep-id' >> ~/.bashrc
+source ~/.bashrc
+```
+
+1. Start the FDO Manufacturer Sample as per the steps outlined in [Manufacturer README](manufacturer/README.md).
+
+2. Complete Device Initialization (DI) by starting the FDO HTTP Java Device Sample as per the steps outlined in [Device README](device/README.md). Delete any existing 'credentials.bin' before starting the device.
```

Comment:
Is it credentials.bin or credential,bin?

Reply:
credentials.bin